azure/cosmos/user.py

```diff
@@ -33,6 +34,7 @@
 
 from .permission import Permission
 
+
```

Comment:
nit: extra new line?

Reply:
recommendation is 2 new lines before class declaration